src/chainstate/coordinator/tests.rs

```diff
@@ -467,8 +467,15 @@ fn test_simple_setup() {
     }
 }
 
-// make it so that we can process the same block in two different pox forks
 #[test]
+// This test should panic until the MARF stability issue
+// https://github.com/blockstack/stacks-blockchain/issues/1805
+// is resolved:
+#[should_panic]
```

Comment:
I've recently noticed that we can be more specific on this kind of tests, by setting an expectation:
```
    #[should_panic(expected = "message from panic!(message)")]
```

Reply:
I agree, but this test is kind of just a placeholder for resolving #1805 -- once #1805 is resolved, this test shouldn't panic at all, so we're not trying to test the specific panic behavior, but rather just have a test in place for when we try to tackle #1805.